integrated_channels/integrated_channel/exporters/content_metadata.py

```diff
@@ -71,24 +71,24 @@ def __init__(self, user, enterprise_configuration):
         super().__init__(user, enterprise_configuration)
         self.enterprise_catalog_api = EnterpriseCatalogApiClient(self.user)
 
-    def _log_info(self, msg):
+    def _log_info(self, msg, course_or_course_run_key=None):
```

Comment:
optional content key to be passed into the formatted logger which will include it as a splunk variable